test/handlers_test.ts

```diff
@@ -97,6 +163,7 @@ describe('taker request handler', () => {
         takerAddress: '0x8a333a18B924554D6e83EF9E9944DE6260f61D3B',
         apiKey: 'kool-api-key',
         canMakerControlSettlement: undefined,
+        comparisonPrice: undefined,
```

Comment:
Was this line necessary?  Seems like it would be better to leave it off.  But I also have seen TypeMoq demand the presence of such things, so wouldn't be surprised if it were required here.

Reply:
yeah, I didn't add this initially, but I had to add this for TypeMoq to be happy O_O